src/niswitch/metadata/enums.py

```diff
@@ -351,6 +352,26 @@
             }
         ]
     },
+    'RelayContact': {
```

Comment:
surprised this change but I don't see it affecting public API so ok

Reply:
Yeah, unused enums are ignored by codegen.